src/gwc/src/main/java/org/geoserver/gwc/config/GeoserverXMLResourceProvider.java

```diff
@@ -49,7 +49,7 @@ public GeoserverXMLResourceProvider(
             throws ConfigurationException {
         this.configFileName = configFileName;
         this.configDirectory = inferConfigDirectory(resourceStore, providedConfigDirectory);
-        LOGGER.info(
+        LOGGER.debug(
```

Comment:
```suggestion
        LOGGER.config(
```

Reply:
Sadly I tried that, but this is a query method which is called multiple times. So the log ends up with the same message multiple times.

Reply:
Actually this was is okay, adds a little noise in the logs but does not repeat; changed to config logging level.